File: demo-vk-rest-scraping.java/src/main/java/demo/model/ObjectsList.java
package demo.model;

import com.google.api.client.util.Key;
import demo.reader.RepeatableReader;

import java.util.List;

public class ObjectsList implements RepeatableReader.Responsive<List<Object>> {

    @Key("response")
    private List<Object> response;

    @Override
    public List<Object> response() {
        return response;
    }

    public void setResponse(List<Object> response) {
        this.response = response;
    }

    @Override
    public String toString() {
        return "ObjectsList{" + "response.size=" + (response != null ? response.size() : "?") + '}';
    }
}

File: demo-vk-rest-scraping.kotlin/src/main/kotlin/demo/model/ObjectsList.kt
package demo.model

import com.google.api.client.util.Key
import demo.reader.RepeatableReader

class ObjectsList : RepeatableReader.Responsive<List<Any>> {

    @Key("response")
    private var response: List<Any>? = null

    override fun response(): List<Any>? {
        return response
    }

    override fun toString(): String {
        return "ObjectsList{" + "response.size=" + (if (response != null) response!!.size else "?") + '}'
    }
}
